<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#552a59;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins SemiBold</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string>ByteCraft Team</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>50</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins SemiBold</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string>Control Panel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="scanCamera">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>130</width>
      <height>130</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:25px;
background-color:#91BDFF;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="scanCamera2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:0;
background-color:#91BDFF;
border-image:url(Data/InterfaceData/qr-code.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="scanNoCamera2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>120</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:0;
background-color:#91BDFF;
border-image:url(Data/InterfaceData/no-camera.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="scanNoCamera">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>100</y>
      <width>130</width>
      <height>130</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:25px;
background-color:#91BDFF;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>410</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins SemiBold</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string>Made By</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>430</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins SemiBold</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#f74d4d;</string>
    </property>
    <property name="text">
     <string>Ouassim Hamdani</string>
    </property>
   </widget>
   <widget class="QPushButton" name="studentList2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>120</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:0;
background-color:#91BDFF;
border-image:url(Data/InterfaceData/student.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="studentList">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>100</y>
      <width>130</width>
      <height>130</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:25px;
background-color:#91BDFF;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>490</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(Data/InterfaceData/logo.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>studentList</zorder>
   <zorder>scanNoCamera</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>scanCamera</zorder>
   <zorder>scanCamera2</zorder>
   <zorder>scanNoCamera2</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>studentList2</zorder>
   <zorder>pushButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../../../../Downloads/test.qrc"/>
 </resources>
 <connections/>
</ui>
